src/code/code_80069420.c

```diff
@@ -1,6 +1,6 @@
 #include "global.h"
 
-void* MemCopy(void* dest, void* src, s32 size) {
+void* MemCopy(void* dest, void* src, ptrdiff_t size) {
```

Comment:
I don't think we should pretend that this function's argument type makes sense: it's no more correct as a ptrdiff than an s32.